libbpf-tools/Makefile

```diff
@@ -2,14 +2,14 @@
 OUTPUT := .output
 CLANG ?= clang
 LLVM_STRIP ?= llvm-strip
-BPFTOOL ?= bin/bpftool
+ARCH := $(shell uname -m | sed 's/x86_64/x86/' | sed 's/aarch64/arm64/' | sed 's/ppc64le/powerpc/')
+BPFTOOL ?= bin/$(ARCH)/bpftool
```

Comment:
using ARCH here might not be a good idea (consider x86 and x86_64, 64-bit binary won't run on 32-bit OS). So let's just use the result of `uname -m` without the "unification" substitutions?